WordPress/src/main/java/org/wordpress/android/ui/AddQuickPressShortcutActivity.java

```diff
@@ -76,17 +76,17 @@ private void displayAccounts() {
         ListView listView = (ListView) findViewById(android.R.id.list);
 
         ImageView iv = new ImageView(this);
-        iv.setBackgroundDrawable(getResources().getDrawable(R.drawable.list_divider));
+        iv.setBackgroundResource(R.drawable.list_divider);
```

Comment:
I was just changing out the call when writing the code.

However reviewing it now, I think the better course of action would be to change this from an `ImageView` to just a `View` and leaving that call the same. This would improve accessibility since the view wouldn't be an `ImageView` without a content description.

Reply:
Actually we might be able to even just call `setFooterDividersEnabled` since this divider view seems to be the same divider that is used within the list. I'll confirm this is the case and commit the result.

Reply:
Changed this to a `View` since `setFooterDividersEnabled` turns out to be for dividers between footers.

Reply:
Sounds good 👍